deploy/helm/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: swo-k8s-collector
-version: 2.2.0-alpha.2
+version: 2.2.0-alpha.3
```

Comment:
Let's revert this version change for now.

Reply:
Reverted